hugo/layouts/report.html

```diff
@@ -294,18 +314,20 @@ <h2 class="mb-3">Your Home's Top Characteristics</h2>
 
   <!-- Comp map -->
   <h2 class="mb-3">Top 5 Most Important Sales</h2>
-  <p>Tap or click on a home to see more details.</p>
+  <p>
+    <i class="bi bi-hand-index-thumb" aria-hidden="true"></i> Tap or click on a home to see more details
+  </p>
   <div
-    id="map-{{ if .is_multicard }}{{ .card.card_num }}{{ else }}main{{ end }}"
+    id="map-{{ .card.card_num }}"
```

Comment:
While revisiting this code, I realized that the use of "main" as the suffix identifier here and for the char table in the single-card case wasn't really necessary, since we still know the card number even in the single-card case. It also makes it more complicated to define JavaScript logic that operates on the map or the table, because we always have to remember to include a separate conditional for single-card PINs compared to multi-card PINs. I didn't end up adding particularly complex JavaScript code in this PR, but I decided to keep this change in case we ever need to do so in the future.